main/acle.md

```diff
@@ -272,6 +272,11 @@ Armv8.4-A [[ARMARMv84]](#ARMARMv84). Support is added for the Dot Product intrin
 * Added a description of support levels in [Current Status and
   Anticipated Changes](#current-status-and-anticipated-changes).
 * Support added for [Function Multi Versioning](#function-multi-versioning).
+* The sections [AES extension](#aes-extension), [SHA2
+  extension](#sha2-extension) and [SHA512
+  extension](#sha512-extension) have been reworded for clarity, by
+  assing the `FEAT_*` tag they refer to from the Arm Architectural
```

Comment:
typo here? I'm not sure what this word is supposed to be

Reply:
ack